sources/packages/api/src/route-controllers/msfaa-integration/msfaa-integration.system.controller.ts

```diff
@@ -0,0 +1,53 @@
+import { Controller, Post } from "@nestjs/common";
+import { MSFAARequestResultDto } from "./models/msfaa-file-result.dto";
+import { InjectLogger } from "../../common";
+import { LoggerService } from "../../logger/logger.service";
+import { AllowAuthorizedParty } from "../../auth/decorators";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { OfferingIntensity } from "../../database/entities";
+import { MSFAARequestService } from "../../msfaa-integration/msfaa-request.service";
+
+@AllowAuthorizedParty(AuthorizedParties.formsFlowBPM)
+@Controller("system-access/msfaa-integration")
+export class MSFAAIntegrationController {
+  constructor(private readonly msfaaRequestService: MSFAARequestService) {}
+
+  /**
+   * Identifies all the records where the MSFAA number
+   * is not requested i.e. has date_requested=null
+   * create a fixed file and send file to the sftp server
+   * to request the MSFAA number
```

Comment:
I believe that "request the MSFAA number" is not accurate. We are the ones generating the number and sending it, right?